### PR TITLE
[No GBP] Removes cogbar from some stealthy actions

### LINKED ISSUE #82583: pickpocketing and tying someone else's shoelaces shows a cog above your head
Reporting client version: 515.1630

## Issue Summary
pickpocketing always does
tying shoelaces seems to be showing the cog, but only some times ???
both should be stealthy actions

## Round ID:
[226993](https://scrubby.melonmesa.com/round/226993)

## Testmerges:
- [Cleans up the SS13_base lua file and adds a new lua file for easily handling multiple signals on different objects.](https://www.github.com/tgstation/tgstation/pull/82458)
- [Admin Verb Datums MkIII | Now with functional command bar](https://www.github.com/tgstation/tgstation/pull/82511)

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Issue brought some missed hidden actions to my attention. 

I left cogbars in for _breaking_ handcuffs because resisting is sort of a gray area. On one hand, you don't want someone to see you doing it; on the other, there is a visible warning that you started doing it. So, meet in the the middle, breaking handcuffs is still visible while resisting isn't.
## Why It's Good For The Game
Closes #82583
Cogbars are not intended to ruin stealth
## Changelog
:cl:
fix: Deviants buffed: Rogue shoelacing, pickpocketing and restraint resisting no longer give cogbar icons.
/:cl:

### CODE REVIEW

File: code/modules/clothing/shoes/_shoes.dm [hunk 1]
```diff
@@ -174,8 +174,7 @@
 				adjust_laces(SHOES_UNTIED, user)
 
 	else // if they're someone else's shoes, go knot-wards
-		var/mob/living/L = user
```
Comment (jlsnow301) [inline code/modules/clothing/shoes/_shoes.dm:177]: this is defined earlier in the proc

**Comment by atlasle** [comment]
great changelog

**Comment by Admiral-Gigaton** [comment]
dont forget about resisting from seats and nest holders (xeno structures)

**Comment by jlsnow301** [comment]
> dont forget about resisting from seats and nest holders (xeno structures)

done

**Pull request merged**